epochStart/bootstrap/epochStartMetaBlockProcessor.go

```diff
@@ -128,7 +128,12 @@ func (e *epochStartMetaBlockProcessor) Save(data process.InterceptedData, fromCo
 		return nil
 	}
 
-	metaBlock := interceptedHdr.HeaderHandler().(*block.MetaBlock)
+	metaBlock, ok := interceptedHdr.HeaderHandler().(*block.MetaBlock)
+	if !ok {
+		log.Warn("saving epoch start meta block error", "error", epochStart.ErrWrongTypeAssertion)
```

Comment:
you could also display the `interceptedHdr.HeaderHandler()` for debugging purposes.

Reply:
done